ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2183,7 +2183,7 @@ presubmits:
         projected:
           sources:
           - secret:
-              name: cluster-secrets-openstack
+              name: cluster-secrets-openstack-vexxhost
```

Comment:
is this going to be a problem? this change was generated by the make jobs command. But I believe the clsuter-secrets-opesntack-vexxhost secret exists, we just won't use it because we are using self manage credentials.

Reply:
There is no problem. https://github.com/openshift/release/pull/16272 sets up the correct secret based on cluster_type.